src/lwip/qpg6100/lwipopts.h

```diff
@@ -89,11 +89,11 @@
 #define MEMP_SEPARATE_POOLS (1)
 #define LWIP_PBUF_FROM_CUSTOM_POOLS (0)
 #define MEMP_USE_CUSTOM_POOLS (0)
-#define PBUF_POOL_SIZE (8)
+#define PBUF_POOL_SIZE (4)
 #define PBUF_POOL_BUFSIZE (1500)
-#define PBUF_POOL_SIZE_LARGE (3)
-#define PBUF_POOL_SIZE_MEDIUM (4)
-#define PBUF_POOL_SIZE_SMALL (5)
+#define PBUF_POOL_SIZE_LARGE (1)
+#define PBUF_POOL_SIZE_MEDIUM (2)
+#define PBUF_POOL_SIZE_SMALL (2)
```

Comment:
What are these additional small/medium/large macros?  They appear in several of our `lwipopts.h` files, don't seem to be used by anything.